vertexai/snippets/src/main/java/vertexai/gemini/GetTokenCount.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vertexai.gemini;
+
+// [START aiplatform_gemini_token_count]
+
+import com.google.cloud.vertexai.VertexAI;
+import com.google.cloud.vertexai.api.CountTokensResponse;
+import com.google.cloud.vertexai.generativeai.preview.GenerativeModel;
+import java.io.IOException;
+
+public class GetTokenCount {
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-google-cloud-project-id";
+    String location = "us-central1";
+    String modelName = "gemini-pro-vision";
+
+    String textPrompt = "How many tokens are there in this prompt?";
+    getTokenCount(projectId, location, modelName, textPrompt);
+  }
+
+  public static int getTokenCount(String projectId, String location, String modelName,
+                                  String textPrompt)
+      throws IOException {
+    try (VertexAI vertexAI = new VertexAI(projectId, location)) {
+      GenerativeModel model = new GenerativeModel(modelName, vertexAI);
+      CountTokensResponse response = model.countTokens(textPrompt);
+
+      int tokenCount = response.getTotalTokens();
+      System.out.println("There are " + tokenCount + " tokens in the prompt.");
+
+      return tokenCount;
+    }
+  }
+}
+// [END aiplatform_gemini_token_count]
```

Comment:
nit: I recommend to follow code snippet guidelines and to narrow the area of the region tag to the `getTokenCount` function only. If you do, please enclose the relevant `import` statements with another pair of the _same_ region tags.